<div class="jumbotron">
  <div class="container">
    <%= form_with(model: user, local: true) do |form| %>
      <% if user.errors.any? %>
        <div id="error_explanation">
          <h2><%= pluralize(user.errors.count,  "件") %>  のエラーがあります。</h2>

          <ul>
          <% user.errors.full_messages.each do |message| %>
            <li><%= message %></li>
          <% end %>
          </ul>
        </div>
      <% end %>
      <h1>New User</h1>

        <div class="field">
          <%= form.label :name %>
          <%= form.text_field :name, id: :user_name %>
        </div>

        <div class="field">
          <%= form.label :email %>
          <%= form.text_field :email, id: :user_email %>
        </div>

        <div class="field">
          <%= form.label :password %>
          <%= form.password_field :password %>
        </div>

        <div class="field">
          <%= form.label :password_confirmation %>
          <%= form.password_field :password_confirmation %>
        </div>

        <div class="field">
          <%= form.label :image %>
          <%= form.file_field :image, id: :user_image %>
          <%= form.hidden_field :image_cache %>
        </div>

        <div class="actions">
          <%= form.submit %>
        </div>
    </div>
  </div>
<% end %>
